lib/rules/number.js

```diff
@@ -2,6 +2,15 @@ import SyncRule from '../types/syncRule';
 
 class NumberValidator extends SyncRule {
 
+    coerce(value) {
+        if (typeof value !== 'string') {
+            return undefined;
+        }
+
+        const parsed = Number(String(value));
```

Comment:
I wouldn't think so since there's already a check higher up that would ensure it being a string.